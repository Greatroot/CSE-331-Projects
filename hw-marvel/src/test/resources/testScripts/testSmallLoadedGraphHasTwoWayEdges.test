# Tests that a small loaded graph has all the correct two way edges.

LoadGraph smallGraph staffSuperheroes.tsv
ListNodes smallGraph
ListChildren smallGraph Ernst-the-Bicycling-Wizard
ListChildren smallGraph Notkin-of-the-Superhuman-Beard