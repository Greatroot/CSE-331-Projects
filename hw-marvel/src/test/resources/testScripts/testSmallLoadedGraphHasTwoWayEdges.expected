# Tests that a small loaded graph has all the correct two way edges.

loaded graph smallGraph
smallGraph contains: Ernst-the-Bicycling-Wizard	Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard Perkins-the-Magical-Singing-Instructor
the children of Ernst-the-Bicycling-Wizard in smallGraph are: Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Notkin-of-the-Superhuman-Beard(CSE403) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Notkin-of-the-Superhuman-Beard in smallGraph are: Ernst-the-Bicycling-Wizard(CSE331) Ernst-the-Bicycling-Wizard(CSE403) Grossman-the-Youngest-of-them-all(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)